sql/character-set-configuration.md

```diff
@@ -6,6 +6,6 @@ category: user guide
 
 # Character Set Configuration
 
-Currently, TiDB only supports the `utf8` character set, which is the equivalent to `utf8mb4` in MySQL. Since MySQL 5.7 defaults to `latin1`, this difference is documented under [default differences](../sql/mysql-compatibility.md#default-differences) between TiDB and MySQL.
+Currently, TiDB only supports the `utf8mb4` character set, and its alias `utf8`. Since MySQL 5.7 defaults to `latin1`, this difference is documented under [default differences](../sql/mysql-compatibility.md#default-differences) between TiDB and MySQL.
```

Comment:
Not your copy, but I would get rid of "Currently".
"and its alias `utf8`" should be changed: in MySQL they are not aliases. I would probably state:
TiDB only supports the `utf8mb4` character set. `utf8` is treated as an alias of `utf8mb4` and is accepted as the recorded character set in some instances. However, in general TiDB will ignore other character sets and record utf8mb4 as the character set being used.

Reply:
Agree, I'll change it to improve readability.